jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketDiagnosticsCollector.java

```diff
@@ -310,4 +312,10 @@ private boolean isWrapper(String valueClass) {
 
         return wsEndpoint;
     }
+
+
+    private String createParamTypeDiagMsg(Set<String> methodParamOptTypes) {
+        String paramMessage = String.join("\n- ", methodParamOptTypes);
+        return "Invalid parameter type. Parameter must be of type: \n- " + paramMessage + "\n- annotated with @PathParams and of type String or any Java primitive type or boxed version thereof";
```

Comment:
Might be a good idea to take the two parts of this message and make them constants.

Reply:
will do that, thanks!